testsuite/forge-cli/src/main.rs

```diff
@@ -1647,12 +1647,26 @@ fn realistic_network_tuned_for_throughput_test() -> ForgeConfig {
         // as no useful transaction reach their mempool.
         // something to potentially improve upon.
         // So having VFNs for all validators
-        .with_initial_fullnode_count(12)
+        /* temp disable VFNs .with_initial_fullnode_count(12) */
         .add_network_test(MultiRegionNetworkEmulationTest::default())
         .with_emit_job(EmitJobRequest::default().mode(EmitJobMode::MaxLoad {
-            mempool_backlog: 150000,
+            mempool_backlog: 500_000,
         }))
         .with_node_helm_config_fn(Arc::new(move |helm_values| {
+            // ask for more resources than normal
+            helm_values["validator"]["resources"]["limits"]["cpu"] = 48.into();
+            helm_values["validator"]["resources"]["requests"]["cpu"] = 48.into();
+            helm_values["validator"]["resources"]["limits"]["memory"] = "72Gi".into();
+            helm_values["validator"]["resources"]["requests"]["memory"] = "72Gi".into();
+            helm_values["fullnode"]["resources"]["limits"]["cpu"] = 48.into();
+            helm_values["fullnode"]["resources"]["requests"]["cpu"] = 48.into();
+            helm_values["fullnode"]["resources"]["limits"]["memory"] = "72Gi".into();
+            helm_values["fullnode"]["resources"]["requests"]["memory"] = "72Gi".into();
+
+            // higher concurrency level
+            helm_values["validator"]["config"]["execution"]["concurrency_level"] = 32.into();
```

Comment:
Note: state sync is in execution mode, so it probably makes sense to set this for the fullnodes, too?